docs/deprecated/pv-recycler.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docwithnav
+title: Persistent Storage Recycler Deprecation
+
+assignees:
+- thockin
+- childsb
+- erinboyd
+---
+
+## Feature description to be deprecated:
+
+When a user creates a persistent volume, they can specify the volume be 'recycled' and reused for a different container or user.
+The use case was driven by the fact 'users' did not have the permissions to create persistent volumes, only an admin can, and therefore
```

Comment:
You can probably remove the single quotes around users to make it consistent with how admin is used. Otherwise, use code style for both of them to distinguish them as literals.